source/docs/software/wpilib-tools/choreo/index.rst

```diff
@@ -0,0 +1,14 @@
+# Choreo
+
+.. image:: /docs/software/wpilib-tools/choreo/images/choreo.png
+
+Choreo is a tool for calculating time optimal autonomous trajectories. This allows simple and efficient creation of autonomous routines.
+
+In Visual Studio Code, press :kbd:`Ctrl+Shift+P` and type ``WPILib`` or click the WPILib logo in the top right to launch the WPILib Command Palette. Select :guilabel:`Start Tool`, then select :guilabel:`Choreo`. Then open your existing `.chor` or save your project to `src/main/deploy`.
```

Comment:
I don't think you need the last sentence, that should be covered in the choreo documentation, and isn't necessarily useful for someone who hasn't used it before